examples/ipfs-from-typed-js/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "ipfs-from-ts",
+  "private": true,
+  "dependencies": {
+    "ipfs": "^0.50.1"
```

Comment:
This needs to be in sync with the version in `/packages/ipfs/package.json` otherwise it may trip that [lerna bug that deletes project files](https://github.com/lerna/lerna/issues/2468#issuecomment-697647195)